chrome_manifest_v3/app.js

```
@@ -4,604 +4,593 @@
+  var text1 = (text || '').trim();
+  var text1lc = text1.toLowerCase();
+  var href1 = (href || '').trim();
+  var href1lc = href1.toLowerCase();
+  var comment1 = (comment || '').trim();
+
+  var retn = '';
+
+  if (text1.length < 1 && href1.length < 1) {
+    // Intetionally blank
+  } else if (text1lc === href1lc) {
+    retn = ' <' + href1 + '> '; // This renders correctly in Trello as a sole URL
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)